core/ui_class_names.js

```diff
@@ -28,4 +28,11 @@ UIClassNames.INNER_CHAT_PANEL_CLASS = "_33LGR"; // the innermost chat panel clas
 UIClassNames.CHAT_ENTRY_CLASS = "_3m_Xw"; // the chat entry in the chats list
 UIClassNames.UNREAD_MARKER_CLASS = "_3cOAM" // the class inside the top-level item;
 
+UIClassNames.CHAT_MESSAGE = "_2wUmf"
+UIClassNames.CHAT_MESSAGE2 = "_21bY5"
```

Comment:
Make it clear what's the difference between `CHAT_MESSAGE`/`CHAT_MESSAGE2` and `DELETED_MESSAGE`/`DELETED_MESSAGE2`, so that in the future when this changes it'll be easier to re-find?

Reply:
done ^^